### PR TITLE
UI - Customer Management Component

### PR DESCRIPTION
**Pull request by RafaelMeha**
Created a new component "customer-management-component", and added the route in the "app.route.ts" file.

### CODE REVIEW

**Review by pmzpro — APPROVED** [review]

File: src/app/customer-management/customer-management.component.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div id="container">
+    <h3 id="page-title">Customer Management</h3>
+    <section id="customers">
+      <button class="btn btn-primary" id="button-add-customer" type="button"><i class="bi bi-plus"></i>Add Customer</button>
+      <table class="table">
+        <tbody>
+          <tr>
+            <td>John James</td>
+            <td><button id="button-view-customer" type="button" class="btn btn-secondary float-end">View</button></td>
```
Comment (Silva096) [inline src/app/customer-management/customer-management.component.html:9]: button-view-customer in css, isn't needed.
You just need btn-light class from bootstrap if i'm not mistaken

File: src/app/customer-management/customer-management.component.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#container{
+    padding: 1rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+}
+
+#page-title{
```
Comment (Silva096) [inline src/app/customer-management/customer-management.component.css:8]: Where is this being used?

File: src/app/customer-management/customer-management.component.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<div id="container">
+    <h3 id="page-title">Customer Management</h3>
```
Comment (Silva096) [inline src/app/customer-management/customer-management.component.html:2]: Class text-center from bootstrap 
Does it work?

File: src/app/customer-management/customer-management.component.html (hunk 1 above)
Comment (RafaelMeha) [reply src/app/customer-management/customer-management.component.html]: fixed!

File: src/app/customer-management/customer-management.component.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    <section id="customers">
+      <button class="btn btn-primary" id="button-add-customer" type="button"><i class="bi bi-plus"></i>Add Customer</button>
```
Comment (Silva096) [inline src/app/customer-management/customer-management.component.html:4]: class "w-100" from bootstrap i believe

File: src/app/customer-management/customer-management.component.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-customer-management',
+  standalone: true,
```
Comment (Silva096) [inline src/app/customer-management/customer-management.component.ts:5]: remove standalone true and imports pls.

**Review by Silva096 — CHANGES_REQUESTED** [review]
Remove css and try to use bootstrap classes as much as possible please

**Review by Silva096 — APPROVED** [review]

**Pull request merged**